rust/main/agents/relayer/src/msg/pending_message.rs

```diff
@@ -271,8 +275,10 @@ impl PendingOperation for PendingMessage {
             }
         };
 
+        let metadata_build_params =
+            MessageMetadataBuildParams::new(ism_address, ISM_MAX_DEPTH, ISM_MAX_COUNT);
```

Comment:
It looks like this is the only place where we construct params. In other places, we copy the options. I wonder why we cannot pass `ISM_MAX_DEPTH` and `ISM_MAX_COUNT` directly into the place we use them instead of passing through all the hierarchy of builders?

Reply:
I think this is a bit more future proof. Where maybe in the future, depth and count is chain-specific.

Reply:
I think it's safe to assume they'll always be global, since the point is to prevent DoS-ing the relayer which could be attempted on any chain / ISM